test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/OpaqueUpgradeTests.cs

```diff
@@ -17,6 +17,38 @@ namespace Microsoft.AspNetCore.Server.HttpSys
 {
     public class OpaqueUpgradeTests
     {
+        [ConditionalFact]
+        public async Task OpaqueUpgrade_DownLevel_FeatureIsAbsent()
+        {
+            if (ComNetOS.IsWin8orLater)
+            {
+                // Skip, this test is for Win7 / 2008R2
```

Comment:
Should you use the [`OSSkipConditionalAttribute`](https://github.com/aspnet/Testing/blob/dev/src/Microsoft.AspNetCore.Testing/xunit/OSSkipConditionAttribute.cs) ?

Reply:
I actually need the opposite, skip except for a specific OS. To use OSSkipConditionalAttribute I'd have to enumerate all possible windows versions, and we don't even have constants for them, we only ever use it to skip Win7/R2.

Reply:
Right. Test Conditions are pretty easy to write though, maybe just write a quick one in this repo to do that? Then it gets properly "Skipped" with a justification message.

Reply:
Just copy-paste-invert the existing one even ;)